docs/example-react-redux.md

```diff
@@ -58,58 +56,68 @@ export function reducer(state = initialState, action) {
 }
 ```
 
-Now here's what your test will look like:
+To test our connected component we can create a custom `render` function using
+the `wrapper` option as explained in the
+[setup](./react-testing-library/setup.md) page.  
+Our custom `render` function can look like this:
 
-```jsx
-// counter.test.js
+```js
+// test-utils.js
 import React from 'react'
+import { render as rtlRender } from '@testing-library/react'
 import { createStore } from 'redux'
 import { Provider } from 'react-redux'
-import { render, fireEvent } from '@testing-library/react'
-import '@testing-library/jest-dom/extend-expect'
-import { initialState, reducer } from './reducer.js'
-import Counter from './counter.js'
-
 
-// this is a handy function that I normally make available for all my tests
-// that deal with connected components.
-// you can provide initialState for the entire store that the ui is rendered with
-function renderWithRedux(
+function render(
   ui,
-  { initialState, store = createStore(reducer, initialState) } = {}
+  {
+    initialState,
+    store = createStore(reducer, initialState),
+    ...renderOptions
+  } = {}
 ) {
-  return {
-    ...render(<Provider store={store}>{ui}</Provider>),
-    // adding `store` to the returned utilities to allow us
-    // to reference it in our tests (just try to avoid using
-    // this to test implementation details).
-    store,
+  function Wrapper({ children }) {
+    return <Provider store={store}>{children}</Provider>
   }
+  return rtlRender(ui, { wrapper: Wrapper, ...renderOptions })
 }
+```
+
+```jsx
+// counter.test.js
+import React from 'react'
+import { createStore } from 'redux'
+import { Provider } from 'react-redux'
+import { fireEvent, screen } from '@testing-library/react'
+// We're using our own custom render function and not RTL's render
+import { render } from './test-utils.js
```

Comment:
In accordance with the setup suggestions, this should be:

```suggestion
// We're using our own custom render function and not RTL's render
// our custom utils also re-export everything from RTL
// so we can import fireEvent and screen here as well
import { render, fireEvent, screen } from './test-utils.js
```

Reply:
Updated it and also added the export to the `test-utils.js` file..
Thanks!